bower.json

```diff
@@ -19,27 +19,22 @@
     "tests"
   ],
   "dependencies": {
-    "alphaville-ui": "Financial-Times/alphaville-ui#^2.6.0",
-    "alphaville-marketslive-chat": "Financial-Times/alphaville-marketslive-chat#^1.0.0",
-    "webchat": "Financial-Times/webchat#^2.1.0",
+    "alphaville-ui": "#c5d9bb6",
```

Comment:
For when [this is released](https://github.com/Financial-Times/alphaville-ui/pull/18):
```suggestion
    "alphaville-ui": "^3.0.0",
```

Reply:
i thought that was a colour at first

Reply:
#c5d9bb duck shell green